pkg/plugins/golang/v3/webhook.go

```diff
@@ -123,7 +123,8 @@ func (p *createWebhookSubcommand) GetScaffolder() (cmdutil.Scaffolder, error) {
 
 	// Create the actual resource from the resource options
 	res := p.resource.NewResource(p.config, false)
-	return scaffolds.NewWebhookScaffolder(p.config, string(bp), res, p.defaulting, p.validation, p.conversion), nil
+	webhook := config.Webhooks{Defaulting: p.defaulting, Validation: p.validation, Conversion: p.conversion}
```

Comment:
`createWebhookSubcommand` should have a [`webhooks config.Webhooks`](https://github.com/kubernetes-sigs/kubebuilder/pull/1696/#discussion_r525435638) field.